modules/repository-datastax/db/mutation/V031__Drop_User_Experiment_Index_table.cql

```diff
@@ -0,0 +1 @@
+drop table user_experiment_index;
```

Comment:
Also, is there anyway to set consistency to ALL while executing these commands or is it something already taken care in the migration-script code (caller of these .cql files).